mixer/adapter/kubernetesenv/config/adapter.kubernetesenv.config.pb.html

```diff
@@ -63,6 +63,18 @@ <h2 id="Params">Params</h2>
 
 <p>Default: 5 minutes</p>
 
+</td>
+</tr>
+<tr id="Params.is_v18_cluster">
+<td><code>isV18Cluster</code></td>
+<td><code>bool</code></td>
+<td>
+<p>Provides support for kubernetes clusters from v1.8.x vintage. Set this to
```

Comment:
I don't think we should track k8s version. API groups are independently versioned, and are sometimes packaged differently for distributions of k8s. It is not guaranteed a certain version of k8s has or has not a particular API version, I believe.